src/App.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react';
-import {SafeAreaView, ScrollView, StyleSheet, Text} from 'react-native';
+import {SafeAreaView, ScrollView, StyleSheet} from 'react-native';
+import IsItCoffeeTime from './components/IsItCoffeTIme';
+import CurrentTime from './components/CurrentTime';
```

Comment:
Small style nit, can we add a line break between third-party module imports and local imports?